config/routes.rb

```diff
@@ -1,10 +1,12 @@
 TmcServer::Application.routes.draw do
+  get 'org/:organization_id/course_templates', to: 'course_templates#list_for_teachers', as: 'list_course_templates'
+  get 'org/:organization_id/course_templates/:id', to: 'course_templates#prepare_course', as: 'prepare_course'
```

Comment:
I was too fast. We need `as: 'prepare_course'` because rails can't create path-helper without it.